spec/views/hq/sponsors/_filters_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe "hq/sponsors/filters.html.haml", type: :view do
+  let(:sponsors_filters) {{
+    "name" => {
```

Comment:
Is this the better way to represent this filter? Cannot we use FactoryGirl for this?